### PR TITLE
Implemented get_analysis_metadata method for Dgraph 

### PR DESCRIPTION
**Pull request by pacospace**

### CODE REVIEW

File: thoth/storages/graph/dgraph.py [hunk 1]
```diff
@@ -214,7 +217,24 @@ def _query_raw(self, query: str, variables: dict = None, *, read_only: bool = Tr
...
+                analyzer_version
+            }   
+        }
+        """ % (IsPartOf.get_label(), analysis_document_id)
+        result = self._query_raw(query)
+
+        if not result:
+            raise NotFoundError(f"Analysis with analysis document if {analysis_document_id} was not found")
+        
+        parsed = result["f"][0]["analysis_datetime"] = parser.parse(result["f"][0]["analysis_datetime"]).replace(tzinfo=timezone.utc)
```
Comment (fridex) [inline thoth/storages/graph/dgraph.py]: Probably without parsed?

```python
        result["f"][0]["analysis_datetime"] = parser.parse(result["f"][0]["analysis_datetime"]).replace(tzinfo=timezone.utc)
```

Comment (pacospace) [reply thoth/storages/graph/dgraph.py]: yes correct! thanks

Comment (pacospace) [reply thoth/storages/graph/dgraph.py]: updated

**Comment by fridex** [comment]
@pacospace please rebase, I tried to do it automatically, but github is complaining...

**Pull request merged**